src/attr/_make.py

```diff
@@ -4,24 +4,11 @@
 import inspect
 import linecache
 import sys
-import threading
 import warnings
 
 from operator import itemgetter
 
-from . import _config, setters
-from ._compat import (
-    HAS_F_STRINGS,
-    PY2,
-    PY310,
-    PYPY,
-    isclass,
-    iteritems,
-    metadata_proxy,
-    new_class,
-    ordered_dict,
-    set_closure_cell,
-)
+from . import _compat, _config, setters
```

Comment:
Sorry, but this is too invasive and adds unnecessary attribute lookups. Please leave the old stuff be and just import `_compat` _additionally_ to it.

Reply:
Done.